src/api/ITooltip.js

```diff
@@ -104,7 +104,7 @@
 
     ITooltip.prototype.tooltipFormat = function (opts) {
         opts = opts || {};
-        opts.label = opts.label || "";
+        opts.label = opts.label || (opts.label.toString && opts.label.toString()) || "";
```

Comment:
What we decided was:
```javascript
opts.label === undefined ? "" : opts.label
```
I think that works as there is an implicit "toString" if needed when writing to the DOM. 

(test and see).

Reply:
@jbrundage bump